source/modules/administration/dictionnaries.rst

```diff
@@ -1,17 +1,14 @@
 Dictionaries
 =============
 
-.. |import-rule| image:: images/importrule.png
-.. |playrule| image:: images/playrule.png
-
 Dictionary's allow to modify data already existing in GLPI or new data in order to group redundant data.
 
 Dictionaries are based on GLPI :doc:`rules engine </modules/administration/rules/rulesmanagement>` and are available for some types of items (software, suppliers, drop-downs). The rules associated with a dictionary will modify values that are either manually inserted or automatically inserted via an inventory tool or via plugins (for instance CSV file injector).
 
 Import, export and duplication
 ------------------------------
 
-Export, import and duplication are available for dictionary's |import-rule|. These operations can be realized globally from dictionary's main page or by batch using mass actions from dictionary's search results. These functionalities are useful for instance when migrating rules from a pre-production environment to a production one.
+Export, import and duplication are available for dictionarys. These operations can be realized globally from dictionary's main page or by batch using mass actions from dictionary's search results. These functionalities are useful for instance when migrating rules from a pre-production environment to a production one.
```

Comment:
i think rules can still be exported and imported

Reply:
They can. All I changed here was a typo (no apostrophe needed) and remove the image of the import and export buttons since they didn't seem needed.

Reply:
can you replace the old image by "or rules"